_posts/wiki/2019-02-15-new-HIRO-member.md

```diff
@@ -23,6 +23,7 @@ The following is a list of things to do if you are new to the {{ site.title }}.
  6. Write an email to [Chantel Lehl](mailto:chantel.lehl@colorado.edu) **with Prof. Alessandro Roncone in Cc**{:.color-banner} in order to get lab access with your Buff card number. Ask her to be added to ECST 322, ECES 116 and ECES 111. Please send her the following information: Name, Student ID, Buff Card Number, CU Login name, Email.
  7. Do a Pull Request [PR] on [this repository](https://github.com/HIRO-group/HIRO-group.github.io) to add your data to [this file](https://github.com/HIRO-group/HIRO-group.github.io/blob/master/_data/people.yml) and a picture of you to [this folder](https://github.com/HIRO-group/HIRO-group.github.io/tree/master/img/people). The picture needs to be square, and preferably not bigger than 400x400 px.
  8. Ask for an invitation to our [Trello](https://trello.com) team and [Mendeley](https://www.mendeley.com) group from **@joewie** on Slack or in the **#hiro-group** channel. Provide your preferred email. We share and keep track of relevant papers on Mendeley, while Trello is where we coordinate work on projects.
+ 9. [Subscribe](https://lists.colorado.edu/sympa/subscribe/hiro-group) to our mailing list! Remember to click on the verification link sent to your email to complete your subscription.
```

Comment:
I would use the whole sentence as a link, i.e. : `[Subscribe to our mailing list!](https://lists.colorado.edu/sympa/subscribe/hiro-group)`

Reply:
Done!